armada_jupyter/submissions.py

```diff
@@ -1,78 +1,32 @@
-from typing import List, Optional, Dict, Union
+from typing import List, Optional
 
 import yaml
+from armada_client.armada.submit_pb2 import IngressConfig, ServiceConfig
+from hikaru import load_full_yaml
+from hikaru.model.rel_1_16 import PodSpec
```

Comment:
This may be a blocker on us using hikaru.  We should not lock ourselves into the Kubernetes 1.16 release (especially since the 1.25 was just released last month).  

We are actively thinking of adopting new features in Kubernetes.  Can you see if there is an option of using a more up to date schema?

Reply:
Hmm. They have up to `https://github.com/haxsaw/hikaru/tree/main/hikaru/model/rel_1_21`. Not ideal.

Reply:
So it looks like they responded to the issue pretty timely but he says he is hoping to get newer releases done.  Since he responded quickly, that is a good sign.  

But I guess the main issue with using OSS code is that the danger of us maintaining is high if the developer decides to stop doing it.  

@jayofdoom What do you think of this?

Reply:
https://github.com/haxsaw/hikaru/graphs/contributors

This looks to be a mostly-one-person-project. This isn't necessarily a problem, but it means we must be willing to take over maintenance of this if it goes away.

Would we be? 
How much effort would that be vs the benefit the library is giving us?

Those are the kinds of questions to think about to answer this :)

Reply:
After discussing in stand-up, we are going to go with a custom podspec object.